tests/test_modeling_rag.py

```diff
@@ -186,10 +186,10 @@ def tearDown(self):
     def get_retriever(self, config):
         dataset = Dataset.from_dict(
             {
-                "id": ["0", "1"],
-                "text": ["foo", "bar"],
-                "title": ["Foo", "Bar"],
-                "embeddings": [np.ones(self.retrieval_vector_size), 2 * np.ones(self.retrieval_vector_size)],
+                "id": ["0", "1", "3"],
+                "text": ["foo", "bar", "qux"],
+                "title": ["Foo", "Bar", "Qux"],
+                "embeddings": [np.ones(self.retrieval_vector_size), 3 * np.ones(self.retrieval_vector_size)],
```

Comment:
This list should have three elements instead of 2

Reply:
Thanks fixed this